net/test/st/run-1x1.sh

```diff
@@ -141,11 +141,11 @@ fi
 		 -s "$MSG_SIZE" \
 		 -E "$CONCURRENCY_SERVER" \
 		 -e "$CONCURRENCY_CLIENT" \
-		 $VERBOSE \
-		 $PARSABLE \
+		 "$VERBOSE" \
+		 "$PARSABLE" \
 		 $BULK_PARAMETERS &
```

Comment:
@rkothiya, Need to ignore the double quote issue at this line, as this was causing 01net ST to fail.

Reply:
done